src/qcodes/instrument/instrument.py

```diff
@@ -56,16 +55,11 @@ class Instrument(InstrumentBase, metaclass=InstrumentMeta):
     _type = None
     _instances: weakref.WeakSet[Instrument] = weakref.WeakSet()
 
-    def __init__(
-        self,
-        name: str,
-        metadata: Mapping[Any, Any] | None = None,
-        label: str | None = None,
-    ) -> None:
+    def __init__(self, name: str, **kwargs: Unpack[InstrumentBaseKWArgs]) -> None:
```

Comment:
This is technically an api break since name and label now cannot be passed as positional arguments. Perhaps we should revert thsi

Reply:
but do you see label/metadata passed positionally inside qcodes or qcodes contrib drivers? if not, then i'd say it's fine (we are still not v1.0, even though, yes, it's good to never break public api)

Reply:
Found one example in contib drivers. I think its fair to fix that and do this https://github.com/QCoDeS/Qcodes_contrib_drivers/pull/330

Reply:
Will fix, merge and do a new contrib driver release and close this

Reply:
tagged qcodes_contrib_drivers v0.22.0 with the fix